src/__snapshots__/parse.ros2idl-autoware-large-message.test.ts.snap

```diff
@@ -7,24 +7,24 @@ Array [
       Object {
         "isComplex": true,
         "name": "header",
-        "type": "std_msgs::msg::Header",
+        "type": "std_msgs/msg::Header",
```

Comment:
This should be `std_msgs/msg/Header` right?

Reply:
why wouldn't this test fail?

Reply:
oh wait, my `.replace` only replaces the first -- updating